scripts/final_adjustment.py

```diff
@@ -36,28 +40,32 @@ def connection_limit_ntc(n,fn,year="2035"):
     
     # adjusting the lines
     for country0, country1 in df.index:
+        original = df.loc[(country0,country1),"s_nom"]
         value = df.loc[(country0,country1),"s_nom_share"]
         if value == 0:
             continue
+        logger.info(f"Set AC transmission limit between {country0} - {country1} from {original} MW to {value} MW")
```

Comment:
Is `value` actually the value that it's being set to? I think we are missing dividing by `s_max_pu`, are we not? @virio-andreyana

Reply:
Value is the total transmission capacity between country0 and country1. the `s_max_pu` is included later in 
`n.lines.loc[lines.index,"s_nom_max"] = value * proportion / n.lines.loc[lines.index,"s_max_pu"]` in the case that  `s_max_pu` happens to be different in different lines

Reply:
I understand. But I don't want to be misleading when specifying the new AC transmission limit. Maybe it makes sense to say that it is an NTC limit? Then it is not implied that that is the actual s_nom that is being set?